shoes-core/spec/shoes/shared_examples/clickable.rb

```diff
@@ -2,4 +2,18 @@
   it { is_expected.to respond_to :click }
   it { is_expected.to respond_to :release }
   it { is_expected.to respond_to :register_click }
+
+  it "can chain click" do
+    result = subject.click do
+    end
```

Comment:
minor thing but maybe `subject.click {}` reads nicer? or even `subject.click do end` - the line break sort of throws me off